raster/r.sun/main.c

```diff
@@ -117,8 +117,10 @@ struct History hist;
 
 int INPUT_part(int offset, double *zmax);
 int OUTGR(void);
+#if 0
 int min(int, int);
 int max(int, int);
+#endif
```

Comment:
This should be handled in a better way. Maybe min and max are always defined "these days" (in this context) or we are missing an include.

Reply:
There is:

https://github.com/OSGeo/grass/blob/7bfe2742618030fc489cb2d3df7800216346a643/include/grass/gis.h#L146-L154

Reply:
Removed this `#if 0` block because `min/max` not used anywhere in this module.